examples/quick_start/scripts/inference/inference.ipynb

```diff
@@ -0,0 +1,530 @@
+{
```

Comment:
<p>I suggest updating this sentence to</p><p>"Please ensure you have followed the <a href="https://github.com/NVIDIA-Merlin/Merlin/blob/main/examples/quick_start/ranking.md" target="_blank">Quick-start for ranking</a> and ran the <code>preprocesssing.py</code> and <code>ranking.py</code> scripts saving the NVTabular preproc workflow and the trained ranking model stored in an accessible location.&nbsp;You also need to follow the instructions at inference README (add link to it)."</p><p> </p><p> </p><p>There is a typo in "Merlin Systems takeS ... and loadS ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>